var/spack/repos/builtin/packages/python/package.py

```diff
@@ -642,6 +643,16 @@ def command(self):
             path = os.path.join(self.prefix.bin, 'python{0}'.format(ver))
             if os.path.exists(path):
                 return Executable(path)
+            else:
+                path = self.prefix.bin.replace(os.sep+"bin","")
+                if sys.platform != "win32":
+                   path = os.path.join(self.prefix.bin, 'python{0}'.format(ver))
+                else:
+                   path = os.path.join(self.prefix, 'python{0}'.format(ver))
```

Comment:
```suggestion
                   path = os.path.join(self.prefix, 'python{0}.exe'.format(ver))
```
Then the next line isn't needed